antarest/study/model.py

```diff
@@ -689,3 +801,62 @@ def to_dto(self) -> MatrixAggregationResultDTO:
 class ReferenceStudy(AntaresBaseModel):
     version: str
     template_name: str
+
+
+class DirectoryMetadata(AntaresBaseModel):
```

Comment:
That module is getting a little messy, at some point we would need at least to separate DB-layer models and plain python models, and probably also to separate them "domaine-wise" (dirs and studies separate)